---a/net/minecraft/block/IGrowable.java
+++b/net/minecraft/block/IGrowable.java
@@ -1,14 +1,16 @@
 package net.minecraft.block;
 
 import java.util.Random;
+
 import net.minecraft.block.state.IBlockState;
 import net.minecraft.util.BlockPos;
 import net.minecraft.world.World;
 
-public interface IGrowable
-{
+public interface IGrowable {
+
     boolean canGrow(World worldIn, BlockPos pos, IBlockState state, boolean isClient);
 
     boolean canUseBonemeal(World worldIn, Random rand, BlockPos pos, IBlockState state);
 
     void grow(World worldIn, Random rand, BlockPos pos, IBlockState state);
-}
+}
